javascript/chronometer.js

```diff
@@ -1,26 +1,63 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = 0
   }
   startClick(callback) {
     // ... your code goes here
+    setInterval(() => {
```

Comment:
VERY IMPORTANT. Every time you create a setInterval you need to save it in a variable, in this case in the intervalId attribute you created in the constructor. This essentially saved the ID of the interval running inside this variable, and it will be the only way to stop it later with the clearInterval function.

Reply:
Doing just this will make the functionality to stop the timer work.
`this.intervalId = setInterval(() => {...`